config/application.rb

```diff
@@ -23,5 +23,7 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+
+    config.active_job.queue_adapter = :sidekiq
```

Comment:
Sidekiq is a job server. Uses Redis as a queue for things to process.

By default, Sidekiq will try to connect to localhost Redis.
We start a docker instance for Redis and forward the port to host => it looks like Redis is running on localhost.